### ISSUE TITLE
Icon Request: VMware Workstation Player

### ISSUE THREAD

**Issue by Ghokun** [mank319/elementaryPlus#339]
Icon name is: vmware-player
Original icon: ![vmware-player](https://cloud.githubusercontent.com/assets/3802058/12290411/5222a6a4-b9ea-11e5-8e65-20ff75e62828.png)

An icon like in installing screen can be made
![vmware](https://cloud.githubusercontent.com/assets/3802058/12290441/80833694-b9ea-11e5-8780-1ba27c0325b0.png)
![vmware2](https://cloud.githubusercontent.com/assets/3802058/12290489/c3d7b83e-b9ea-11e5-8e03-9a059d69e58f.png)

**Comment by elmat0** [comment]
Happy with [this?](https://github.com/elmat0/icons_etc/blob/master/apps/128/vmware-player.svg)

**Comment by rootsandculture** [comment]
Could you create the VMWare Workstation and Virtual Network Editor elementarized icons too?
![screenshot from 2016-01-17 12 16 01](https://cloud.githubusercontent.com/assets/12083250/12378150/5756ff1c-bd14-11e5-9fde-d51bda02e49d.png)
Those three icons are installed when installed the VMWare Workstation software.

**Comment by cybre** [comment]
@rootsandculture He'll need the icon names

**Comment by rootsandculture** [comment]
You means the location? i.e /usr/local/share/applications/vmware-netcfg.desktop ?

**Comment by Ghokun** [comment]
@elmat0 its awesome !

**Comment by cybre** [comment]
@rootsandculture Icon names from the desktop launchers

**Comment by rootsandculture** [comment]
/usr/local/share/applications/vmware-netcfg.desktop
/usr/local/share/applications/vmware-player.desktop
/usr/local/share/applications/vmware-workstation.desktop

**Comment by cybre** [comment]
@rootsandculture No, open those file and you'll see a line like this `Icon=<some-icon-name-here>`
We need that `<some-icon-name-here>`

**Comment by cybre** [comment]
Or just post the output of `cat /usr/local/share/applications/vmware-* | grep Icon=`

**Comment by rootsandculture** [comment]
Icon=vmware-netcfg
Icon=vmware-player
Icon=vmware-workstation

**Comment by rootsandculture** [comment]
http://3.bp.blogspot.com/-C6_1I2VpSNQ/VFpP7Tt5jGI/AAAAAAAAAC8/96kNpDQuBMc/s1600/VMware-Workstation-Icon-10.png

**Comment by rootsandculture** [comment]
I wish help with creating icons, can someone give me an procedure how to do it and software used?.

**Comment by cybre** [comment]
@rootsandculture 
elementary HIG: https://elementary.io/docs/human-interface-guidelines#iconography
You can use Inkscape.
